src/routes/Apm.tsx

```diff
@@ -1,30 +1,59 @@
+import InfoOutlinedIcon from "@mui/icons-material/InfoOutlined";
 import React from "react";
 import { useTranslation } from "react-i18next";
 import { pipe } from "fp-ts/function";
 import * as E from "fp-ts/Either";
 import * as O from "fp-ts/Option";
 import {
+  Box,
+  Button,
   FormControl,
   FormControlLabel,
+  Link,
   Radio,
-  RadioGroup
+  RadioGroup,
```

Comment:
Are we saying that for `apm` onboarding the only supported method will be Paypal only? This is true for now, but what about the future? this route was set to apm to handle all onboardable apm methods (such as Paypal). idk, maybe here can be the case to add this modification only in the case that the selected payment method id is a paypal one. what do you think?
cc @infantesimone

Reply:
correct, in the future we may have other onboardable methods, so it is better not to bind paths with methods

Reply:
we will resume the route APM generalization in a dedicated task cc @vito80ba @ciuffagianluca @pietro-tota